lib/dor/services/client.rb

```diff
@@ -153,6 +153,7 @@ def build_connection(with_retries: false)
           #   the Faraday instance will be passed an empty block, which
           #   causes the adapter not to be set. Thus, everything breaks.
           builder.adapter Faraday.default_adapter
+          builder.options[:timeout] = 300 # 5 minutes timeout for very lage object create/update (default if none set is 60 seconds)
```

Comment:
approving;  one more comment for your consideration:  Andrew said the "largeness" of this object was that it had a ton of files ... that might be good to capture in the comment.  (As opposed to a few large files)